tests/io_components/test_docs.py

```diff
@@ -0,0 +1,13 @@
+import inspect
+
+from kartothek.io.eager import store_dataframes_as_dataset
+from kartothek.io_components.docs import _PARAMETER_MAPPING
+
+
+def test_docs_store_dataframes_as_dataset():
```

Comment:
Could you apply this test to all functions in `kartothek.io` starting with `read_`, `update_` or `store_`?